integration_tests/tests/amplify.rs

```diff
@@ -0,0 +1,14 @@
+#![cfg(feature = "amplify")]
+
+extern crate rusoto_amplify;
+extern crate rusoto_core;
+
+use rusoto_amplify::{Amplify, AmplifyClient};
+use rusoto_core::Region;
+
+#[test]
+fn should_work() {
+    let client = AmplifyClient::new(Region::UsEast1);
+    let response = client.list_apps(Default::default()).sync();
```

Comment:
Can we add a `.expect(...)` to this call as well as the other tests? If the call returns an error this test would still pass.

Reply:
Sure thing. I scanned other tests and it seemed they were mostly printing for smoke test runs. I'll see if I can do a bit more

Reply:
done. the one exception is the apigateway management api which requires an actual endpoint of an apigateway to point at. see the docs in the integration test for more details